### PR TITLE
Add Softmax and Layernorm Subgraph

### PR DESCRIPTION
**Pull request by HertzHan**

### CODE REVIEW

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+class LayerCase(paddle.nn.Layer):
+    def __init__(self):
+        super().__init__()
+        self.variance_epsilon = 1e-6
+        self.reduce_num = 4096
+    def forward(self, x, weight, bias):
+        mean = paddle.sum(x, axis=-1, keepdim = True) / self.reduce_num
+        var = weight / paddle.sqrt((x - mean).pow(2).sum(-1, keepdim = True)/self.reduce_num + self.variance_epsilon)
+        return var * (x - mean) + bias
```
Comment (jiahy0825) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm.py]: 这里不用手动拆分，恢复到调用 paddle layer_norm 的代码，组合算子会做拆分

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            net = PaddleLayernormSubGraph()
+        net.eval()
+        net = utils.apply_to_static(net, use_cinn)
+        for i in range(10000):
+            out = net(self.x, self.weight, self.bias)
+        return out
+
+    def test_train(self):
+        cinn_out = self.train(use_cinn=True)
+        # print("DEBUG: cinn_out = ", cinn_out)
```
Comment (jiahy0825) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm.py]: 删除不必要的 DEBUG 信息

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+import numpy as np
+import utils
+
+import paddle
+
+class LayerCase(paddle.nn.Layer):
+    def __init__(self):
+        super().__init__()
+        # self.variance_epsilon = 1e-6
+        # self.reduce_num = 4096
```
Comment (jiahy0825) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax.py]: 同上，删除这里

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+    def forward(self, x, axis=-1):
+        max_ = paddle.max(x, axis = axis,keepdim = True)
+        output = paddle.exp(x - max_) / paddle.sum(paddle.exp(x - max_), axis = axis, keepdim=True)
+        return output
```
Comment (jiahy0825) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/softmax.py]: 同上

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+import numpy as np
+import utils
+
+import paddle
+
+class LayerCase(paddle.nn.Layer):
+    def __init__(self):
+        super().__init__()
+        self.variance_epsilon = 1e-6
+        self.reduce_num = 4096
```
Comment (jiahy0825) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm.py]: 这个没用到可以删除

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm_fp32_shape_1_13_4096.py [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+import numpy as np
+# import utils
+
+import paddle
+
+class LayerCase(paddle.nn.Layer):
+    def __init__(self):
+        super().__init__()
+    def forward(self, x, weight, bias):
+        out = paddle.nn.functional.layer_norm(x, x.shape[-1],  weight, bias, self.variance_epsilon)
```
Comment (Zeref996) [inline framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm_fp32_shape_1_13_4096.py:18]: self.variance_epsilon 未定义

**Review by Zeref996 — APPROVED** [review]

File: framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm_fp32_shape_1_13_4096.py (hunk 6 above)
Comment (jiahy0825) [reply framework/e2e/PaddleLT_new/layercase/perf_monitor/manual_subgraph/layernorm_fp32_shape_1_13_4096.py:18]: done, thanks~
